docs/standard/linq/creating-xml-trees.md

```diff
@@ -0,0 +1,425 @@
+---
+title: "Creating XML Trees - .NET"
```

Comment:
I'll actually add the suffix to the docfx.json file so let's remove it from here

```suggestion
title: "Create XML Trees"
```